Makefile

```diff
@@ -3,9 +3,7 @@ SHORT_NAME := workflow-manager
 # Enable vendor/ directory support.
 export GO15VENDOREXPERIMENT=1
 
-# SemVer with build information is defined in the SemVer 2 spec, but Docker
-# doesn't allow +, so we use -.
-VERSION := git-$(shell git rev-parse --short HEAD)
+include versioning.mk
```

Comment:
I think you'll need to define `SHORT_NAME` above this include (see line 4-5 of versioning.mk).